src/main/java/io/stargate/sgv2/jsonapi/api/model/command/table/definition/datatype/VectorColumnDesc.java

```diff
@@ -46,7 +47,7 @@ public boolean equals(Object o) {
       return false;
     }
     var vectDesc = (VectorColumnDesc) o;
-    return dimensions == vectDesc.dimensions
+    return Objects.equals(dimensions, vectDesc.dimensions)
```

Comment:
double check for Objects.equals and hash when one of the values is null

Reply:
Confirmed that `Objects.equals(a, b)` return false if one of the values is null:

1. Returns `true` if `a` and `b` are both null.
2. Returns `false` if one is `null` and the other is not.
3. If neither is `null`, it calls `a.equals(b)`.

Update the unit test in `ObjectMapperConfigurationTest.AlterTable.addColumns()` to cover the null case:
```
assertThat(columns)
.containsEntry(
    "vector_1",
    new VectorColumnDesc(
        null,
        new VectorizeConfig("nvidia", "NV-Embed-QA", null, null)));
```
The `assertThat(columns).containsEntry(...)` assertion relies on the equals() method of the objects being compared.